packages/builders/src/interactions/slashCommands/options/integer.ts

```diff
@@ -17,7 +17,7 @@ export class SlashCommandIntegerOption
 	/**
 	 * {@inheritDoc ApplicationCommandNumericOptionMinMaxValueMixin.setMaxValue}
 	 */
-	public setMaxValue(max: number): this {
+	public setMaxValue(max: number | null): this {
 		numberValidator.parse(max);
 
 		Reflect.set(this, 'max_value', max);
```

Comment:
This is missing `?? undefined`, as well in several more places.

Reply:
When `.setNameLocalizations(null)` is called on main, it sets that to `null`, not `undefined`, so shouldn't it be as so here?

https://github.com/RedGuy12/discord.js/blob/b1797a655a7647aa8959d393e84829214b1d8bd2/packages/builders/src/interactions/slashCommands/mixins/NameAndDescription.ts#L70-L74

Reply:
@kyranet could I get a response on this?

Reply:
Sorry for the delay, I had a very busy week.

This is different, `max_value` and `min_value` are defined as `?: number` (`?: number | undefined` fits better and matches the current behaviour, to be honest):

https://github.com/discordjs/discord.js/blob/edadb9fe5dfd9ff51a3cfc9b25cb242d3f9f5241/packages/builders/src/interactions/slashCommands/mixins/ApplicationCommandNumericOptionMinMaxValueMixin.ts#L2-L4

If `null` can be set here, then the types for those two properties must also be changed.

And I believe that would cause a type mismatch, see the types for [`APIApplicationCommandIntegerOptionBase`](https://github.com/discordjs/discord-api-types/blob/2dc3fd0a159266b37415b8316a31509153c78c27/payloads/v10/_interactions/_applicationCommands/_chatInput/integer.ts#L8-L18).